File: src/main/java/no/nav/melosys/eessi/service/eux/BucSearch.java
package no.nav.melosys.eessi.service.eux;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BucSearch {

    private String fnr;
    private String fornavn;
    private String etternavn;
    private String foedselsdato;
    private String rinaSaksnummer;
    private String bucType;
    private String status;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/eux/BucSearch.kt
package no.nav.melosys.eessi.service.eux

data class BucSearch(
    var fnr: String? = null,
    var fornavn: String? = null,
    var etternavn: String? = null,
    var foedselsdato: String? = null,
    var rinaSaksnummer: String? = null,
    var bucType: String? = null,
    var status: String? = null
)
